components/03-modules/gallery/gallery.config.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  title: 'Product grid item',
+  context: {
+    class: '',
+    productLabel: {
+      text: 'NEW',
+      type: 'new'
+    },
+    typeHorizontal: false,
+    imageMain: {
+      src: '/images/gallery/product-img-496-496.jpg'
```

Comment:
change `src` to `defaultSrc` b/c `src` is 'reserved' (?) for lazyloading

Reply:
done